### PR TITLE
Enhancement: Py3 compatability

### PR DESCRIPTION
**Pull request by Mr-TaylorHobbs**

### CODE REVIEW

File: forms_ext/views/generic.py [hunk 1]
```diff
@@ -17,6 +11,9 @@ class MessageFormMixin(object):
     """
 
     def form_valid(self, form):
+        print(type(form))
```
Comment (khornberg) [inline forms_ext/views/generic.py]: prints

File: setup.py [hunk 2]
```diff
@@ -1,80 +1,29 @@
...
+    description="Extensions for Django's Forms.",
+    long_description=open('README.txt', 'r').read(),
+    url="http://github.com/imtapps/django-forms-ext",
+    packages=find_packages(exclude=["example"]),
+    install_requires=open('requirements/dist.txt').read().split("\n"),
+    tests_requires=open('requirements/dev.txt').read().split("\n"),
+    test_suite='runtests.runtests',
+    zip_safe=False,
+    classifiers=[
+        "Development Status :: 3 - Alpha",
```
Comment (khornberg) [inline setup.py]: Is this still alpha?

**Comment by Mr-TaylorHobbs** [comment]
Remove lettuce because the upgrade path to python 3 would be extremely lengthy and the tests fail it python 2. If there would be benefit in these tests, we will have to bring them back somehow.

File: setup.py [hunk 3]
```diff
@@ -1,80 +1,28 @@
...
-    def run(self):
-        os.system("pip install ./")
-        os.system("pip install %s" % self.get_test_dependencies())
-
-if __name__ == '__main__':
-    do_setup()
+setup(
+    name="django-forms-ext",
+    version=VERSION,
+    author="Aaron Madison & Matthew J. Morrison",
```
Comment (mattjmorrison-imt) [inline setup.py]: Change the author to "IMT Computer Services" I think.... also, I would get rid of the VERSION constant and just drop the version in the setup.py directly like we do for everything else.

**Pull request merged**